### PR TITLE
used ChatCompletion to create a baseline for the new approach

### PR DESCRIPTION
**Pull request by epinzur**

### CODE REVIEW

File: ChatCompletion_v1.ipynb [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# re-run this cell as often as needed until the process is completed\n",
+    "\n",
+    "from IPython.display import clear_output\n",
+    "\n",
+    "with open(\"ChatCompletion_v1_results.jsonl\", \"a\") as out:\n",
```
Comment (kerinin) [inline ChatCompletion_v1.ipynb:402]: Could this be run from the output directly, rather than saving to JSON files first?

**Review by kerinin — APPROVED** [review]

File: ChatCompletion_v1.ipynb (hunk 1 above)
Comment (epinzur) [reply ChatCompletion_v1.ipynb:402]: its not saving to json first... its only writing out to json after receiving the response from the model.

Comment (kerinin) [reply ChatCompletion_v1.ipynb:402]: Sorry - I meant the conversion to pandas, ie `for row in conversations_df.iterrows():`

Comment (epinzur) [reply ChatCompletion_v1.ipynb:402]: ah yes.  I can make that update.

**Pull request merged**